### DISCUSSION TITLE
Need help, running podman on windows

### DISCUSSION THREAD

**Discussion by ChauhanPriyanka** [containers/podman#23850] [General]
Hi, I'm new to podman, need help to fix the following issue - **bash: line 1: ./build.sh: Permission denied**

I'm running following command
```
podman run -it --name docker_builder_img --log-level=debug -v C:\Users\PriyankaChauhan\.ssh:/root/.ssh/ -e USR=$USERNAME -e PSW=$PASSWORD -e VERSION=3.000 docker_builder_img :latest
```

I'm getting following error 

```
level=debug msg="DoRequest Method: POST URI: http://d/v5.2.2/libpod/containers/f54d5492d24460644426989be03f100f2b9dbabb8765c2333d0a6ea58ecd35d7/start"
bash: line 1: ./build.sh: Permission denied
```

I checked build.sh permissions, it has full access.
Also it is present in same directory with Dockerfile.

Please help to identify the root cause.

**Comment by rhatdan** [comment]
Can you add a :Z to your -v volume mount
podman run -it --name docker_builder_img --log-level=debug -v C:\Users\PriyankaChauhan\.ssh:/root/.ssh/`:Z` -e USR=$USERNAME -e PSW=$PASSWORD -e VERSION=3.000 docker_builder_img :latest
